library/src/main/java/com/fstyle/library/GravityEnum.java

```diff
@@ -0,0 +1,42 @@
+package com.fstyle.library;
+
+import android.annotation.SuppressLint;
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.view.Gravity;
+import android.view.View;
+
+public enum GravityEnum {
+  START,
+  CENTER,
+  END;
+
+  private static final boolean HAS_RTL =
+      Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1;
+
+  @SuppressLint("RtlHardcoded")
+  public int getGravityInt() {
+    switch (this) {
+      case START:
+        return HAS_RTL ? Gravity.START : Gravity.LEFT;
```

Comment:
ERROR: 'block' child have incorrect indentation level 8, expected level should be 16.